src/NextGen/Gateways/Stripe/NextGenStripeGateway/NextGenStripeGateway.php

```diff
@@ -70,27 +67,17 @@ public function enqueueScript(): EnqueueScript
 
     /**
      * @since 0.1.0
-     * @throws ApiErrorException
      */
     public function formSettings(int $formId): array
     {
         $this->setUpStripeAppInfo($formId);
 
         $stripePublishableKey = $this->getStripePublishableKey($formId);
         $stripeConnectedAccountKey = $this->getStripeConnectedAccountKey($formId);
-        $currency = give_get_currency($formId);
-        $formDefaultAmount = give_get_default_form_amount($formId);
-        $defaultAmount = Money::fromDecimal(!empty($formDefaultAmount) ? $formDefaultAmount : '50', $currency);
-        $stripePaymentIntent = $this->generateStripePaymentIntent(
-            $stripeConnectedAccountKey,
-            $defaultAmount
-        );
 
         return [
             'stripeKey' => $stripePublishableKey,
-            'stripeClientSecret' => $stripePaymentIntent->client_secret,
-            'stripeConnectedAccountKey' => $stripeConnectedAccountKey,
-            'stripePaymentIntentId' => $stripePaymentIntent->id,
+            'stripeConnectedAccountId' => $stripeConnectedAccountKey,
```

Comment:
Yeah, that is **so** much easier not having the generate a payment intent upfront - awesome.

Reply:
yup!